lib/ansible/module_utils/gcp_utils.py

```diff
@@ -76,7 +76,12 @@ def __init__(self, module, product):
     def get(self, url, body=None, **kwargs):
         kwargs.update({'json': body, 'headers': self._headers()})
         try:
-            return self.session().get(url, **kwargs)
+            # Ignore the google-auth library warning for user credentials. More
+            # details: https://github.com/googleapis/google-auth-library-python/issues/271
+            import warnings
```

Comment:
Note there probably isn't a reason to import warnings here rather than at the toplevel.

Reply:
@abadger only reason is that I thought that it probably won't be used anywhere else. Would you like me to move it to the toplevel? I can make a PR in devel about it? Or are you thinking to do that?

Reply:
I'm afraid I don't touch lib/ansible code in devel anymore.  Yeah, if you want to just fix it in devel, that would be fine with me.